docs/user/adding-pdf-annotations.md

```diff
@@ -317,16 +317,23 @@ you can use the {py:class}`Highlight <pypdf.annotations.Highlight>`:
 ```python
 from pypdf import PdfReader, PdfWriter
 from pypdf.annotations import Highlight
+from pypdf.generic import ArrayObject
+from pypdf.generic._base import FloatObject
 
 pdf_path = os.path.join(RESOURCE_ROOT, "crazyones.pdf")
 reader = PdfReader(pdf_path)
 page = reader.pages[0]
 writer = PdfWriter()
 writer.add_page(page)
 
+rect = (50, 550, 200, 650)
+quad_points = [rect[0], rect[1], rect[2], rect[1], rect[0], rect[3], rect[2], rect[3]]
+
 # Add the highlight
 annotation = Highlight(
-    vertices=[(50, 550), (200, 650), (70, 750), (50, 700)],
+    rect=rect
```

Comment:
This seems to be a syntax error, as it misses a comma.

Reply:
Yeah, now I added the comma